ui/src/components/dashboard/components/charts/custom/Bar.vue

```diff
@@ -18,16 +18,17 @@
     import {Bar} from "vue-chartjs";
 
     import {customBarLegend} from "../legend.js";
-    import {useTheme} from "../../../../../utils/utils";
-    import {defaultConfig, getConsistentHEXColor,} from "../../../../../utils/charts.js";
+    import {useTheme} from "../../../../../utils/utils.js";
+    import {defaultConfig, getConsistentHEXColor, chartClick} from "../../../../../utils/charts.js";
```

Comment:
This allows to convert those files in the future to typescript, without changing all files.

```suggestion
    import {useTheme} from "../../../../../utils/utils";
    import {defaultConfig, getConsistentHEXColor, chartClick} from "../../../../../utils/charts";
```